src/Markdown.js

```diff
@@ -62,6 +62,9 @@ function is_multi_line(node) {
  */
 export default class Markdown {
     constructor(input) {
+        // Support GH-style strikeout
+        input = input.replace(/~~(.*?)~~/g, '<del>$1</del>');
```

Comment:
I thought we'd decided to not support this? Although I don't fully understand what context this is being use din